mimesis/providers/text.py

```diff
@@ -134,11 +134,14 @@ def hex_color(self) -> str:
         :return: Hex color code.
 
         :Example:
-            #D8346B
+            #d8346b
         """
-        letters = '0123456789ABCDEF'
-        color_code = '#' + ''.join(self.random.sample(letters, 6))
-        return color_code
+        return '#{:x}'.format(
```

Comment:
Seems like there is an issue, because sometimes the length of the returned string is 6, but must be 7.